rs/tests/nested/src/util.rs

```diff
@@ -45,6 +60,106 @@ pub(crate) fn check_hostos_version(node: &NestedVm) -> String {
     s.trim().to_string()
 }
 
+/// Submit a proposal to elect a new GuestOS version
```

Comment:
These might exist as utils in other tests, but these are nice and small, so that de-duplication can wait if it is not convenient.

Reply:
I looked around, but couldn't find anything. Specifically, I looked at occurrences of `submit_update_elected_replica_versions_proposal` and `submit_update_unassigned_node_version_proposal`. They use it directly without any helper. I leave it at that.